### DISCUSSION TITLE
Week 5 post

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#111] [General]
I'm posting a little prematurely (I haven't finished all the assignments yet) because I'm probably not going to be able to post later this week. If I'm able to, I'll come back and edit this post. 

With Flexbox Froggy, level 24 last week didn't show the yellow frogs for me. This week when I clicked the link to the game, it took me to level 24 completed with the yellow frogs. I don't know if the game changes or what, but it shows I completed it even though I didn't do anything with yellow frogs last week.

I'm struggling through JavaScript still. I'm feeling like a broken record at this point, but the instructions aren't super helpful. I will try to use more of the helps within the program this week to see how that works out.

I think I could use HTML to create a basic website/webpage for a couple of my projects: herbal website or instructional/lesson website. I could get a title down on my own and build from there. The Cafe and Registration forms assignments helped me understand how to begin creating content on a website. With some googling to help me remember how exactly to code certain elements, I think I could probably create a webpage on my own, which is really cool. I don't know where to start though as far as actually creating a website URL to then be able to begin coding, so I hope to learn that eventually.

**Comment by metasean** [comment]
> I'm struggling through JavaScript still.  

Please, remember that I have office hours.  Sometimes it's easier just to have a conversation and work through examples together.

> I think I could probably create a webpage on my own, which is really cool. 

🚀 🎉

>  I don't know where to start though as far as actually creating a website URL to then be able to begin coding, so I hope to learn that eventually.

You'll be learning how to start your own projects shortly.  We won't cover the details of website hosting (e.g. using your own personal domain), but most of you should be able to use GitHub itself to host/publish your website, and that will come with it's own url (typically something like `https://user-name.github.io/project-name/`)

**Reply by Skye404** [reply]
@metasean I thought I replied, but I guess I didn't. Your office hours aren't at great times for me. But I'll keep that in mind and might attend this Saturday if I can make it. If it's possible to have an office hour either Monday afternoon or Tuesday/Wednesday/Friday morning those would be regular times I would probably attend each week.

**Reply by metasean** [reply]
- Wed, June 21st @ 10am
- Thu, June 29th @ 10am

[June Calendar](https://usu.instructure.com/calendar#view_name=month&view_start=2023-06-17)

**Reply by Skye404** [reply]
Thanks! I'll plan to attend Wednesday at 10am.
